Classes/Mvc/RequestBuilder.php

```diff
@@ -29,17 +29,16 @@
 
 use Helhum\TyposcriptRendering\Exception;
 
-/**
- * Class RequestBuilder
- */
 class RequestBuilder
 {
     /**
-     * @param array $rawRequestArgument
+     * @param mixed[] $rawRequestArgument
+     *
      * @return Request
+     *
      * @throws \Helhum\TyposcriptRendering\Exception
      */
-    public function build($rawRequestArgument)
+    public function build(array $rawRequestArgument)
```

Comment:
OK, checked the code. This is done on purpose.

Your code change will lead to a fatal error, which will inevitably always bubble up to the top.
My code throws a dedicated exception, which can bubble up to the top, but can also be caught and handled correctly if desired.

But you are right that the type annotation should rather be mixed then